yarn-project/circuit-types/src/simulation_error.ts

```diff
@@ -93,7 +95,7 @@ export class SimulationError extends Error {
          * @returns The stack.
          */
         get() {
-          return this.getStack();
```

Comment:
👍 
Still curios to find out why `getStack` is undefined here.

Reply:
The `this` here is just wrong! it's the this of the stack object, not class

Reply:
You can tell because it has type `any` in TS on hover in IDE

Reply:
Hmm, I don't think that's it. The spec says `this` is going to be the object the property is accessed on. This works as expected:

```js
let obj = { getStack() { return "foo"; } }
undefined
Object.defineProperty(obj, "stack", {
  get() {
    return this.getStack();
  }
});
Object { getStack: getStack(), … }

obj.stack 
"foo" 
```